pyartcd/pyartcd/pipelines/release_from_fbc.py

```
@@ -414,6 +426,128 @@ async def _load_mr_approvers_from_group_config(self) -> dict[str, list[str]]:
+                '--load-disabled',
+                'images:print',
+                '--show-non-release',
+                '--short',
+                '{component}: {name}',
+            ]
+            _, output, _ = await exectools.cmd_gather_async(cmd)
+            for line in output.strip().splitlines():
+                parts = line.split(': ', 1)
+                if len(parts) == 2 and parts[0].strip() == component:
+                    return parts[1].strip()
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)